ix-dev/community/monerod/questions.yaml

```
@@ -87,6 +87,21 @@ questions:
             max: 65535
             required: true
             show_if: [["tor_connections_enabled", "=", true]]
+        - variable: tor_inbound_connections_enabled
+          label: Enable inbound Tor connections
+          schema:
+            type: boolean
+            default: false
+            show_if: [["tor_connections_enabled", "=", true]]
+        - variable: tor_inbound_address
+          label: Inbound onion address
+          description: |
+            Port needs to be configured under "Network".
+          schema:
+            type: string
+            required: true
+            valid_chars: "^[a-zA-Z0-9]*.onion$"
```

```suggestion
            valid_chars: "^[a-zA-Z0-9]*\.onion$"
```
I assume the `.` is literal here right?

---

Yes. Thanks!